tests/integration/test_metrics.py

```diff
@@ -1241,35 +1200,40 @@ def test_generic_metric_extraction(mini_sentry, relay):
             }
         ],
     }
+    config["transactionMetrics"] = {"version": 1}
+    config["dynamicSampling"] = {
+        "rules": [],
+        "rulesV2": [
+            {
+                "id": 1,
+                "samplingValue": {"type": "sampleRate", "value": 0.0},
+                "type": "transaction",
+                "condition": {"op": "and", "inner": []},
+            }
+        ],
+    }
 
     transaction = generate_transaction_item()
     timestamp = datetime.now(tz=timezone.utc)
     transaction["timestamp"] = timestamp.isoformat()
     transaction["start_timestamp"] = (timestamp - timedelta(seconds=2)).isoformat()
 
-    # Explicitly test a chain of Relays
-    relay = relay(relay(mini_sentry, options=TEST_CONFIG), options=TEST_CONFIG)
+    relay = relay(mini_sentry, options=TEST_CONFIG)
     relay.send_transaction(PROJECT_ID, transaction)
 
     envelope = mini_sentry.captured_events.get(timeout=3)
-    assert envelope.items[0].headers.get("type") == "transaction"
-    assert envelope.items[0].headers.get("metrics_extracted") is True
```

Comment:
We can't do this anymore as we drop the envelope in order to extract metrics

Reply:
Would be good to negate the assertion here, i.e. none of the items have type "transaction".